packages/authgear-web/src/container.ts

```diff
@@ -425,6 +425,9 @@ export class WebContainer {
     if (options?.uiLocales != null) {
       q.set("ui_locales", options.uiLocales.join(" "));
     }
+    if (options?.closeOnSuccess) {
+      q.set("close_on_success", "true");
```

Comment:
Actually on web, we can set the redirect URI to `javascript:window.close()` to close the tab.